zOSMF-Samples/discovery.txt

```diff
@@ -0,0 +1,785 @@
+/*              workflow will be set to disabled as well.            */
+/*                                                                   */
+/*              When all is said and done the workflow itself will   */
+/*              run through its conditions with the new variables    */
+/*              and will change the state of its steps accordingly.  */
+/*              (See z/OSMF documentation for more details on this)  */
+/*                                                                   */
+/*              IMPORTANT: 1. The variable names defined in this exec*/
+/*                         must match the same name of the variables */
+/*                         defined in the workflow definition file   */
+/*                         or it won't work properly.                */
+/*                                                                   */
+/*                         2. The file directory for $output is      */
+/*                         defined within the workflow definition    */
+/*                         file, NOT this exec.                      */
+/*                                                                   */
+/*                                                                   */
+/* CREATE DATE: 06/03/15                                             */
+/*                                                                   */
+/* AUTHORS: Keith Miller, Jeff Bland                                 */
+/*                                                                   */
+/* CHANGE ACTIVITY:                                                  */
+/*                                                                   */
+/* FLAG  PERSON  DATE       REASON                                   */
+/* ----  ------  ---------  ---------------------------------------- */
+/*  N/A  KMILLER 06/03/15   Initial Version                          */
+/*  1703 BLAND   03/20/17   Validate Program number first            */
+/*  1804 BLAND   04/04/18   Test for upper case Z/OS NAME            */ 
+/*********************************************************************/
+/*                                                                   */
+/*********************************************************************/
+/*                         MAIN PROGRAM                              */
+/*********************************************************************/
+address "TSO"
+xrc = 0
+name = 'CONDIST'
+/*-------------------------------------------------------------------*/
+/*    Check the current state of priced feature on user's system.    */
+/*    using either the D PROD,REG or D PROD,STATE command.           */
+/*-------------------------------------------------------------------*/
+/* runcmd = '${inputcmd}' */
+/*
+if runcmd = 'Use D PROD,STATE' then do
+  command = 'D PROD,STATE'
+end
+else if runcmd = 'Use D PROD,REG' then do
+  command = 'D PROD,REG'
+end
+*/
+command = 'D PROD,STATE'

```

Comment:
@shreeji818  The discovery.txt is identical as the file used in z/OS upgrade workflow. I got Marna's agreement to publish it. I would suggest keeping the block in the sample so that workflow users can switch to “D PROD,REG” command if needed. Thanks.